package.json

```diff
@@ -6,14 +6,14 @@
     "build": "run-p type-check build-only",
     "preview": "vite preview",
     "test:unit": "vitest --environment jsdom --root src/ --watch=false",
-    "test:unit:dev": "vitest --environment jsdom --root src/",
+    "test:unit:dev": "cross-env-shell OVERWRITE_TAPES=$OVERWRITE_TAPES \"vitest --environment jsdom --root src/\"",
```

Comment:
I'm wondering if it's worth using https://www.npmjs.com/package/@naholyr/cross-env for default values, but I guess if it's not set then it won't overwrite tapes so it's probably fine then :smile:

Reply:
What I was trying to overcome with that is allowing env vars passed into `npm run` to be further passed into whatever the npm script is. I've not had to do this before, and I was initially drawn to cross-env to allow env bar to be handled on any OS. Is there maybe a better way to do this? I agree that adding a dependency in for just this is overkill. 😅